### ISSUE TITLE
Binaries for PyTorch 1.9.0

### ISSUE THREAD

**Issue by dimzog** [facebookresearch/detectron2#3158]
## 🚀 Feature
Pre built binaries for new PyTorch version (1.9.0) and CUDA 11.1

## Motivation & Examples

I'm having problems working with 3xxx nvidia series and cuda 11.1.
I rolled back to torch 1.7.1 and cuda 10.2, but VRAM caching seems to be a tremendous issue.

Hopefully, new releases fix that.

**Comment by ParadoxRobotics** [comment]
Hi, 
Did you build Detectron 2 from source ?
What kind of issue ? 
I personally have have a core dump when I import anything from Detectron 2.

**Comment by dimzog** [comment]
I did not build from source, as wheels fails to bundle everything. What i suspect is nvcc version of the 3070.
My 1060ti works just fine, so the issue must be with the 3xxx series.

What i end up using in 3070 is
`pip3 install torch==1.7.1+cu110 torchvision==0.8.2+cu110 torchaudio===0.7.2 -f https://download.pytorch.org/whl/torch_stable.html`

and 

`python3 -m pip install detectron2 -f https://dl.fbaipublicfiles.com/detectron2/wheels/cu110/torch1.7/index.html`

which runs, but VRAM quickly fills up.

Issues I'm having are similar to https://github.com/facebookresearch/detectron2/issues/2837 and https://github.com/pytorch/pytorch/issues/55027

**Comment by dimzog** [comment]
I managed to fix the 3xxx issues. Posting incase anyone needs this info.

What i did was follow the tutorial for installing CuDNN 8.2 (link below), but

- Instead of CUDA 11.2 Toolkit, search the archives and find CUDA 11.3 that matches your driver version.
- Instead of CuDNN 8.1, install CuDNN 8.2 (23/4/2021)

`https://medium.com/analytics-vidhya/install-cuda-11-2-cudnn-8-1-0-and-python-3-9-on-rtx3090-for-deep-learning-fcf96c95f7a1`

I followed instructions provided in the link, until conda installation. I do not run conda, therefore it is of no use to me.

Then i simply installed detectron2 from source. Everything seems to work like a charm for the time.
